apps/frontend/static_src/scss/variables.scss

```diff
@@ -0,0 +1,6 @@
+// Wagtail Guide conf.
+$enable-rounded: false;
+$green:   teal !default;
+$primary: $green;
+
+$body-bg: #EDEAF3;
```

Comment:
Nitpick, follow the convention of Bootstrap with lower case values.

We might look into a linter for frontend code. (for a later PR)

Reply:
I didn't understand this one. Can you point me to that section of bootstrap docs?